Homework_2/Program.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Homework_2.UserActions;
+
+namespace Homework_2
+{
+    internal class Program
+    {
+        enum UserActions
```

Comment:
enums are usually stored in separate file

Reply:
++